### DISCUSSION TITLE
Generative AI Disclosures

### DISCUSSION THREAD

**Discussion by nathangathright** [Podcastindex-org/podcast-namespace#663] [Ideas]
In light of the PSP’s blog post, the most recent Podcasting 2.0, and past discussions (#458, #531, #602), it seems like there’s renewed interest in disclosing the use of generative AI. In my opinion, a simple Boolean flag encourages listeners to write off a podcast as “AI slop” without much nuance. Instead, what if we introduced a tag to disclose the software used to produce an episode?

```xml
<podcast:aiDisclosure tool="AutoPod" purpose="editing"/>
<podcast:aiDisclosure tool="Descript" purpose="transcription"/>
<podcast:aiDisclosure tool="AIVA" purpose="music"/>
<podcast:aiDisclosure tool="Resemble AI" purpose="synthetic_voice"/>
<podcast:aiDisclosure tool="Dall-E" purpose="episode_artwork"/>
<podcast:aiDisclosure tool="ChatGPT" purpose="script_writing"/>
<podcast:aiDisclosure tool="Google Translate" purpose="translation"/>
```

Following the pattern of the person tag, using this tag at the channel level would indicate the default tools used. Using this tag at the item level **replaces** the disclosures for that episode.

Apps would be under no obligation to display all this information. 
* Some might choose to display a “nutrition label” like Adobe’s [Content Credentials](https://contentcredentials.org/).
* Some might only check for `purpose="transcription"` to display a message in their transcript view like, “This transcript was generated automatically. Its accuracy may vary.”

I’m unsure how to handle shows that only use generative AI for an alternate enclosure. For instance, a podcast might provide translated audio with synthesized voices.
```xml
<enclosure url="..."/>
<podcast:alternateEnclosure url="" lang="es">
  <podcast:source uri="..." />
</podcast:alternateEnclosure>
<podcast:transcript url="..." language="en" />
<podcast:transcript url="..." language="es" />
<podcast:aiDisclosure tool="Google Translate" purpose="translation"/>
<podcast:aiDisclosure tool="Resemble AI" purpose="synthetic_voice"/>
```
I’d love to hear others’ thoughts on the best way to associate one of these tags with a given media asset. I considered adding a `type` attribute to point to media with a matching MIME type, but I’m unsure if that’s the best approach.

Ultimately, **I’m not convinced there is much utility to listeners**, but I wanted to explore the possibilities and identify any interesting use cases that might generate momentum. I suspect there would be a lot more demand for a tag allowing podcasters to express their feelings about using their content for AI training and inference, even without an enforcement mechanism.
```xml
<podcast:aiUsage ai-training="notAllowed" ai-inference="allowed">
```

**Comment by johnspurlock** [comment]
> Ultimately, I’m not convinced there is much utility to listeners

As a listener, my main interest would be to identify (and exclude) shows in search that used synthetic voices.  Perhaps all of these tools aren't necessary then, and we could get away with a standardized attribute on the existing `podcast:person` tag

**Comment by samsethi** [comment]
Hi Nathan / John

I like the idea of labelling and we do need to show if a show is sponsored or has advertising.  Spotify did this recently and I suspect more podcasts will need to declare their advertising/sponsor status as a label.  I also like the option to show what software generated the AI. We would add this to the podcast infobox in TrueFans. 

In TrueFans we have already extended the Podcast Taxonomy to include AI as a host in the person tag.  BTW we have also added a secondary role in the person tag as people can be both a host and producer or singer and songwriter and these roles are hyperlinked so you can find other producers or songwriters etc but I digress. 

The way we are using the person=AI will be to offer users a toggle switch in their user settings to hide/disable listening to episodes where the host is AI which hI think what John wants. 

We already do this for episodes tagged Explicit.  We disable the play button and if the episode is in a playlist we skip the episode.  I think the AI toggle switch in the user settings could work the same way. 

We are also extending our flag option.  Today you can report a comment or clip with a flag but now we are going to add an option to flag content as AI.  We will still need to review the flag before we approve the tag internally. 

Having podcast creators self label their content and tag the host as AI in their RSS feed would be the best option.  

<img width="500" alt="Fullscreen_05_10_2024__12_25" src="https://github.com/user-attachments/assets/9f6a2019-8947-4351-ad40-2579919d357f">

**Reply by jamescridland** [reply]
Writing scripts using AI isn't covered by this use-case

**Comment by jamescridland** [comment]
Use of AI isn't a boolean value. So, at the very least, I'd like to ask for a percentage here.

If I was supporting this tag, Podnews Daily [on Thursday](https://podnews.net/update/apple-transcripts-to-8) (there's a listen button below the main image) would have had this in there:

```
<podcast:aiDisclosure tool="Wondercraft" purpose="synthetic_voice" percentage="7"/>
```

... i.e. a very small amount of that show (7% by total duration) had a synthetic voice in it (near the end of the main news section). It didn't, otherwise, use a synthetic voice.

@johnspurlock has already indicated that he would use this feature to "hide podcasts that used synthetic voices". Yet, I don't believe that he wants to remove something like a short clip, like I've used here. Why should I be penalised for using a tiny amount of AI in a responsible manner?

**Comment by samsethi** [comment]
Hi James 

I am trying to make this simple. The (co)host is either AI or not AI so a switch saying if host/person = AI then hide the episode is all I want to offer as v1.0. We do the same with the Explicit tag.  The episode is either explicit or not explicit.  We don't have a percentage of explicit as a measure, thus it is boolean.  

However, going forward we could be more nuanced.  A percentage of AI as a measure of the amount of AI used in the episode e.g clip might help listeners but I am not sure podcasters will mark their episode as partial AI.  And even if they do mark it 10% or 50% how are apps supposed to use this labelling?

**Comment by johnspurlock** [comment]
Perhaps a simple way of handling that would be to add a `speakerTime` attribute to the episode-level person tag.  This would be useful for a variety of things including weighing generated vs human speech, and now with transcription is fairly easy to come up with.

**Reply by jamescridland** [reply]
Of a hypothetical episode, there's a small portion of the script that was written by AI.

```
<podcast:aiDisclosure tool="ChatGPT" purpose="script_writing" percentage="15" />
```

I'm unclear how this gets integrated in a person tag.

**Reply by johnspurlock** [reply]
ah I see I have waded into a nutrition-facts-style aiDisclosure proposal.  I'm not sure this makes sense, or if anyone would do it (except if regulation forced them to), or if it would be useful (who cares which tool was used, and sometime soon, who even knows? 
 behind layers of abstractions etc).  I don't even think "ai" is a future-proof term to use in a tag name here, I think we'll look back on it shortly like we do "information superhighway".

I'm for proposals that provide publishers crisp ways of marking up their audio, segments, people involved, with additional attributes useful for podcast apps and services that are only known by the publisher, and have the publishers stamp of authority.  So I like the idea of attributing episodes/chapters/people with attributes like "scripted", "generated", or maybe even "human" and "not-human"

There are several new podcast publishers cranking out entirely-generated shows.  It would be nice for apps to have a filter/section for these, but ultimately I think it's going to be on the client side to identify these, not being able to rely on the publisher side.

Wading out!

**Comment by jamescridland** [comment]
For noting: [Listen Notes has made a Notebook LM Detector](https://github.com/ListenNotes/notebooklm-detector).

**Reply by johnspurlock** [reply]
and he is even maintaining a list!  yeoman's work

https://github.com/ListenNotes/notebooklm-generated-fake-podcasts/blob/main/fake_podcasts.csv

**Comment by geeknews** [comment]
At this time, the AI-generated voice is the primary issue. If we make this too complicated, no one will use it. I do not mind if 10 words are changed in the descript, but I do want flagged content that is all AI-generated voice. To that though what happens when a show uses eleven labs to create a version of the show with their cloned voice or one of those provided by eleven labs.

There should also be a reporting system that allows listeners to flag a show when they suspect it is AI-generated. If we focus on the generated voice section this would be a good start

1. Show Produced in Full with AI Generated voice (Yes/NO)
2. Show Produced in full with AI Cloned Voiced (Yes/NO)
3. Show Produced with AI Voice acting as a host/cohost/guest (Yes/No)
4. Show Edited with AI Cloned Voice (Yes/No)


I think you get these first 3 in your good as a starting point. It should be its own tag

We should have a external way for people to report AI voiced shows as well

**Reply by nathangathright** [reply]
Tagging synthetic voices feels much more feasible than my initial proposal. That’d be enough for podcast apps to display a disclosure message. How did you imagine the reporting mechanism working? Would a listener be notifying their app or the host of the feed?
